app/components/Resources/ResourcesTable.js

```diff
@@ -315,6 +334,31 @@ function buildAddressCell(addresses) {
 	return <span>{addressString}</span>
 }
 
+// Returns the open hours today or null if closed
+function openHours(schedule_days) {
```

Comment:
You're right. I was just using what was there when I started working on the project--I think it was done with an underscore to mimic the style of the property on the object that comes back from the database. I can change it, though. I think I might need to make a bunch of changes anyway once the other pull requests are merged, to fix conflicts.